source/extensions/common/sqlutils/sqlutils.h

```diff
@@ -0,0 +1,18 @@
+#include "common/protobuf/utility.h"
+
+#include "include/sqlparser/SQLParser.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Common {
+namespace SQLUtils {
+
+class SQLutils {
+public:
+  static bool setMetadata(const std::string& query, ProtobufWkt::Struct& metadata);
```

Comment:
It seems like we need to add docs on this.

Reply:
I see what you mean. Actually at this moment nothing changed from metadata point of view. Documentation for mysql is still valid:https://www.envoyproxy.io/docs/envoy/v1.14.1/configuration/listeners/network_filters/mysql_proxy_filter.html?highlight=metadata#dynamic-metadata. 
When I link this `setMetadata` to Postgres filter I will create similar docs describing what metadata is created by Postgres filter, but this will happen once this PR is merged.